tools/sotn_str/jp.py

```diff
@@ -124,6 +124,21 @@ def get_chr(chr):
     return table[chr]
 
 
+alt_table = "".join([
+    #0 1 2 3 4 5 6 7 8 9 A B C D E F
+    "装備技システム短剣必殺使攻撃力防",
+    "御魔導器拳こ一覧棒両手食物爆弾盾",
+    "投射薬ん右左武兜鎧マントその他い",
+])
+
+alt_utf8_to_index = {}
+for index, value in enumerate(alt_table):
+    alt_utf8_to_index[value] = index
+
+for index, value in enumerate("ＡＴＤＥＦ"):
```

Comment:
This is a separate table at D_800A2D64. I think the right approach is having two different macros because this table is only in the US version, and D_800A2D10 etc. are only in the JP version. It's not the same encoding

Reply:
OK, there's are `_S2` and `_S2_HD` macros for the separate lookup tables.